test/plural_resolver_test.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:i18next/i18next.dart';
+import 'package:i18next/src/plural_resolver.dart';
+import 'package:mockito/mockito.dart';
+
+abstract class FakeCheckerKeyFunction {
+  bool call(String pluralModifier);
+}
+
+class MockCheckerKeyFunction extends Mock implements FakeCheckerKeyFunction {}
+
+void main() {
+  group('PluralResolver', () {
+    MockCheckerKeyFunction checkerKeyFnc;
+    PluralResolver subject;
+
+    setUpAll(() {
+      checkerKeyFnc = MockCheckerKeyFunction();
+      subject = PluralResolver();
+    });
+
+    test('when count is equal 1 returns empty string', () {
+      const count = 1;
+      final result = subject.pluralize(
+        count,
+        I18NextOptions.base,
+        checkerKeyFnc,
+      );
+
+      verifyZeroInteractions(checkerKeyFnc);
+      expect(result, equals(''));
+    });
+
+    group('when count is different than 1', () {
```

Comment:
Missing count zero, it is more to keep it explicit what a zero should return or not